app/mailers/rent_mailer.rb

```diff
@@ -4,4 +4,12 @@ def new_rent_send(rent_id)
     locale(@rent.user)
     mail(to: @rent.user.email, subject: t('subject_rent_email'))
   end
+
+  def expire_rent_send
+    rents = Rent.where(end_date: Time.zone.today)
```

Comment:
I would put the logic of searching the rents in the worker, the mailer should only know how to send mails

Reply:
Done